cmd/add.go

```diff
@@ -68,6 +68,10 @@ func add(cmd *cobra.Command, args []string) error {
 		Password:     password,
 	}
 
+	if err := creds.Validate(); err != nil {
+		return fmt.Errorf("Failed to validate credentials: %s", err)
```

Comment:
any reason not to use errfailedtosetcredentials error here?

Reply:
Yeah, I wanted to include the upstream error.

Reply:
I guess that means the ErrFailedToSetCredentials definition is redundant now, so I'll take that out

Reply:
https://github.com/segmentio/aws-okta/pull/56/commits/997fc6691b63ecf8b93529eec7743048d87f0a72

Reply:
oh! i meant adding additional errors. It seems the pattern in this repo is to reference errors instead of creating errors inline. I'm guessing this is useful for others who might want to condition their code on what type of error is returned. Could we add an error like ErrInvalidCredentials and return if validate fails? and for the one below, return the original "ErrFailedToSetCredentials"? sorry for the confusion